media-libs/openh264/openh264-2.1.0.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils multilib-minimal toolchain-funcs
+
+MOZVER=39
+DESCRIPTION="Cisco OpenH264 library and Gecko Media Plugin for Mozilla packages"
+HOMEPAGE="https://www.openh264.org/"
+SRC_URI="https://github.com/cisco/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/mozilla/gmp-api/archive/Firefox${MOZVER}.tar.gz -> gmp-api-Firefox${MOZVER}.tar.gz"
+LICENSE="BSD"
+SLOT="0/5" # subslot = openh264 soname version
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="cpu_flags_x86_avx2 +plugin utils"
+
+RESTRICT="bindist"
+
+BDEPEND="
+	abi_x86_32? ( dev-lang/nasm )
+	abi_x86_64? ( dev-lang/nasm )"
+
+DOCS=( LICENSE CONTRIBUTORS README.md )
+
+PATCHES=( "${FILESDIR}/${PN}-2.1.0-pkgconfig-pathfix.patch" )
+
+src_prepare() {
+	default
+
+	multilib_copy_sources
+}
+
+multilib_src_configure() {
+	ln -s "${WORKDIR}"/gmp-api-Firefox${MOZVER} gmp-api || die
+}
+
+emakecmd() {
+	CC="$(tc-getCC)" CXX="$(tc-getCXX)" LD="$(tc-getLD)" \
+	emake V=Yes CFLAGS_M32="" CFLAGS_M64="" CFLAGS_OPT="" \
+		PREFIX="${EPREFIX}/usr" \
+		LIBDIR_NAME="$(get_libdir)" \
+		SHAREDLIB_DIR="${EPREFIX}/usr/$(get_libdir)" \
+		INCLUDES_DIR="${EPREFIX}/usr/include/${PN}" \
+		HAVE_AVX2=$(usex cpu_flags_x86_avx2 Yes No) \
+		$@
+}
+
+multilib_src_compile() {
+	local mybits="ENABLE64BIT=No"
+	case "${ABI}" in
+		s390x|alpha|*64) mybits="ENABLE64BIT=Yes";;
+	esac
+
+	emakecmd ${mybits} ${tgt}
+	use plugin && emakecmd ${mybits} plugin
+}
+
+multilib_src_install() {
+	emakecmd DESTDIR="${D}" install-shared
+
+	if use utils ; then
+		newbin h264enc openh264enc
+		newbin h264dec openh264dec
+	fi
+
+	if use plugin; then
+		local plugpath="usr/$(get_libdir)/nsbrowser/plugins/gmp-gmp${PN}/system-installed"
+		insinto "/${plugpath}"
+		doins libgmpopenh264.so* gmpopenh264.info
+		echo "MOZ_GMP_PATH=\"${EROOT}${plugpath}\"" >"${T}"/98-moz-gmp-${PN}
```

Comment:
Welcome to EAPI=7! `$EROOT` no longer has a trailing slash (see https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html#d-ed-root-eroot-no-longer-have-a-trailing-slash). So please use absolute paths (for `plugpath` already).